pytorch_lightning/plugins/training_type/deepspeed.py

```diff
@@ -780,13 +783,6 @@ def load_optimizer_state_dict(self, checkpoint: Mapping[str, Any]) -> None:
         # override to do nothing, deepspeed engine already loaded the states in `load_checkpoint_file()`
         pass
 
-    def update_global_step(self, total_batch_idx: int, current_global_step: int) -> int:
```

Comment:
Why did you remove this ? DeepSpeed doesn't perform an update on latest batch of an epoch as Lightning currently does.

Reply:
This is why we introduce `update_global_step` hook.